src/main/java/Display.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Display {
+    public void printGreeting() {
+        System.out.println("Welcome to TicTacToe!");
+    }
+
+    public void promptPlayer(String playerMark) {
+        System.out.println(String.format("Make a move: %s", playerMark));
+    }
+
+    private List currentGridCells(Board board, int size) {
+        List currentGridAsArray = new ArrayList<>();
+        Board boardClass = board;
```

Comment:
I'm confused by this line. The name of the variable doesn't help. Why are we assigning an argument to a temporary variable?

Reply:
✅ unnecessary variables removed PR #6